controller/service.rst

```diff
@@ -66,36 +66,39 @@ Then you can define it as a service as follows:
 
         # app/config/services.yml
         services:
-            app.hello_controller:
+            AppBundle\Controller\HelloController:
                 class: AppBundle\Controller\HelloController
 
     .. code-block:: xml
 
         <!-- app/config/services.xml -->
         <services>
-            <service id="app.hello_controller" class="AppBundle\Controller\HelloController" />
+            <service id="AppBundle\Controller\HelloController" class="AppBundle\Controller\HelloController" />
```

Comment:
With fqcn named services you can use all controller notations (e.g. `AppBundle\Controller\HelloController::fooAction`, `App:Hello:foo`, etc.). No need to define the service if you use `@Route`.

With other service ids you have to use the single colon notation: `app.hello_controller:fooAction`.